### ISSUE TITLE
Feature Request: Multiple "X Marks The Spot" Vehicles

### ISSUE THREAD

**Issue by daveake** [akarpenko/spacenear-nosql#1]
I've used the special "XX" ID to denote an "X marks the spot" vehicle, to show the landing position predicted by my payload.  However I'd like to show this for more than one flight at the same time.  It'd be really nice to be able to have multiple X's on the map - e.g. XX1, XX2, or whatever.

**Comment by SteveRan** [comment]
Yeah - I have started to use this too and having a single X marks the spot vehicle is quite limiting - can we have several at once: XX1, XX2 would be fine as Dave says - or another option if that would make things simpler to implement.
